src/cmap/handshake/handshake_document.ts

```diff
@@ -0,0 +1,18 @@
+import type { Document } from 'bson';
+
+import type { ClientMetadata } from './client_metadata';
+
+/** @internal */
+export interface HandshakeDocument extends Document {
+  /**
+   * @deprecated Use hello instead
+   */
+  ismaster?: boolean | 0 | 1;
+  hello?: boolean | 0 | 1;
```

Comment:
We are careful to make sure these are set to 1 now, it was a v5 change, and also when would they be set to zero?